src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/StartupCode/StartupCodeHelpers.cs

```diff
@@ -18,6 +20,7 @@ internal static class StartupCodeHelpers
         internal static void Initialize()
         {
             InitializeStringTable();
+            RunEagerClassConstructors();
             RuntimeImports.RhEnableShutdownFinalization(0xffffffffu);
```

Comment:
Sure! I'll send another pull request to delete it. I don't want to convolute this one with unrelated things.